graal-tools/src/main/scala/io/cloudstate/graaltools/ProtobufGeneratedMessageRegisterFeature.scala

```diff
@@ -11,9 +11,9 @@ import java.util.concurrent.ConcurrentHashMap
 final class ProtobufGeneratedMessageRegisterFeature extends Feature {
```

Comment:
This should be modified. With the old version of protobuf that Akka used, the `equals`, `hashCode` and `toString` implementations used reflection, which is why every method needed a reflection configuration. But the new protobuf version they're on, when configured to be optimized for speed (rather than bytecode size), generates `equals`, `hashCode` and `toString` methods that don't use reflection. Furthermore, all of the `akka.protobufv3.internal.*` subtypes, Akka never loads them or interacts with them using reflection, rather, Akka has hand coded serializers that convert the Akka types to these types statically, and uses short manifest names, rather than classnames, with static mappings of these names to the protobuf types, when deserializisng. So, we should be able to just remove `akka.protobufv3.*` from here, no reflection configuration should be needed for it.

We still do need some config for the Cloudstate messages that we expect to be serialized by the Akka protobuf serializer, but I think there might be some problems with the code below, the Akka protobuf serializer uses `parseFrom` and `toByteArray` reflectively, but I don't see those being registered. Also, not sure how much of an overhead this is going to be to have configuration for classes that don't need it, we have a *lot* of protobuf classes that don't get loaded reflectively, only those sent over Akka remoting as top level messages need reflection configuration, which is only a handful of messages.

Reply:
@jroper I'll try removing protobufv3 and report back

Reply:
@jroper You were right! Removing the protobufv3 worked. I'll see if I can remove the need for the methods again.

Reply:
@jroper It would be interesting to simplyfy this AutomaticFeature—but the challenge is the ridiculously long turnaround time to try even the smallest changes. Feel free (encouraged!) to find optimizations here! :+1: